build.gradle

```diff
@@ -17,12 +19,11 @@ allprojects {
         google()
         mavenCentral()
     }
-
-    plugins.withId("com.vanniktech.maven.publish") {
+    /*plugins.withId("com.vanniktech.maven.publish") {
```

Comment:
Revert to original

Reply:
I have issues on jitpack.io builds
I will fix it